semana3/blackjack/desafio.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * EXEMPLO DE UTILIZAÇÃO DA 'comprarCarta'
+ * 
+ * 
+    const carta = comprarCarta(); // Sorteia uma carta. Por exemplo, o rei de ouros
+    
+    console.log(carta.texto) // imprime o texto da carta. Exemplo: "K♦️" (indica "K" de ouros)
+    console.log(carta.valor) // imprime o valor da carta (um número). Exemplo: 10 (dado que "K" vale 10)
+ * 
+ * 
+ * 
+ */
+
+console.log("Bem vindo ao jogo de BlackJack!");
+
+if(confirm("Quer iniciar uma nova rodada ?")) {
+   let cartasJogador =[];
+   let cartasComputador = [];
+   let pontuacaoJogador;
+   let pontuacaoComputador;
+
+   for(let i = 0; i < 4; i++) {
+      if(i % 2 === 0) {
+         cartasJogador.push(comprarCarta());
+      } else {
+         cartasComputador.push(comprarCarta());
+      }
+   }
```

Comment:
Excelente uso do for (com o validador de números pares haha). Também era possível fazer com um array com duas iterações que preenchesse os dois de uma vez, mas adorei a prática extra!

Reply:
Muito obrigado pelas palavras! Neste caso eu não pensei em fazer dessa forma. Acho que mentalizei a forma que seria dada num jogo original (uma carta para cada jogador de uma vez kkkk)